code/modules/shuttle/syndicate.dm

```diff
@@ -35,8 +34,8 @@
 	shuttleId = "steel_rain"
 	possible_destinations = null
 
-/obj/machinery/computer/shuttle/syndicate/drop_pod/Topic(href, href_list)
-	if(href_list["move"])
+/obj/machinery/computer/shuttle/syndicate/drop_pod/tgui_act(action, list/params, datum/tgui/ui, datum/tgui_state/state)
+	if(action == "move")
 		if(z != level_name_to_num(CENTCOMM))
 			to_chat(usr, "<span class='warning'>Pods are one way!</span>")
 			return 0
```

Comment:
`0` becomes `FALSE`. Also don't call `..()` like this as I pointed out in my first comment or so.